### PR TITLE
Checkbox Component

### PR DESCRIPTION
**Pull request by oliveira-gust**

### CODE REVIEW

File: libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  public _checked = false;
+  @Input() public set checked (value: any) {
+    this._checked = !!value;
+  }
+
+  /**
+   * Emits an event when user clicks checkbox.
+   * Event contains the id and value of the checkbox.
+   */
+  @Output() public event = new EventEmitter<CheckboxEvent>();
```
Comment (guilhermejcgois) [inline libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts]: This should be bind to a form control

File: libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+  /**
+   * Label text transform.
+   * - up: uppercase
+   * - lo: lowercase
+   * - cp: capitalize
+   * - no: none
+   * @default up
+   */
+  @Input() public labelTransform: 'up' | 'lo' | 'cp' | 'no' = 'up';
```
Comment (guilhermejcgois) [inline libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts]: Label can be passed as it need to be shown

File: libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+   * - red: #df5e3d
+   * - darkblue: #314a61
+   * - blue: #815ae8
+   * - lightblue: #0dd6f4
+   * - yellow: #fcbd1f
+   * - pink: #ed4581
+   * - gradient: from #10c0c6 to #4dde90
+   * @default gradient
+   */
+  @Input() public backgroundColor: 'red' | 'darkblue' | 'blue' | 'lightblue' | 'yellow' | 'pink' | 'gradient' = 'gradient';
```
Comment (guilhermejcgois) [inline libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts]: This color need to be set by material theme, not standalone

File: libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts (hunk 1 above)
Comment (guilhermejcgois) [reply libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts]: or mat-checkbox change event

File: libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+  /**
+   * Checkbox size.
+   * - sm: small (12px)
+   * - md: medium (14px)
+   * - lg: large (16px)
+   *
+   * @default 'sm'
+   */
+  @Input() public size: 'sm' | 'md' | 'lg' = 'sm';
```
Comment (guilhermejcgois) [inline libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts]: font size can be set by container, you can inspect in https://material.angular.io/components/checkbox/examples and change `mat-card-content` font-size to see changes

**Comment by oliveira-gust** [comment]
Styling for checkbox will be done in another project

**Pull request closed without merging**